packages/hook-sparql-endpoint-comunica/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jbr-hook/sparql-endpoint-comunica",
-  "version": "2.6.0",
+  "version": "3.6.0",
```

Comment:
I guess this change was done by mistake?

Reply:
Indeed, same as above, not sure how that happened